components/desktop/mate/mate-session-manager/Makefile

```diff
@@ -0,0 +1,73 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Till Wegmueller
+# Copyright 2016 Ken Mays
+#
+
+include ../../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		mate-session-manager
+COMPONENT_VERSION=	1.14.0
+COMPONENT_PROJECT_URL=	http://www.mate-desktop.org
+COMPONENT_SUMMARY=	The Session manager and session startup stuff
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:190b9d9d2d9e76b8aa8ef84f351aab07fbea7e0772cecc11dae8c441c67b8dde
+COMPONENT_ARCHIVE_URL=	http://pub.mate-desktop.org/releases/1.14/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE= GPLv2, LGPLv2, FDLv1.1
+COMPONENT_LICENSE_FILE= $(COMPONENT_NAME).license
+COMPONENT_CLASSIFICATION = System/Libraries
+COMPONENT_FMRI = library/desktop/mate/$(COMPONENT_NAME)
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+PATH=/usr/gnu/bin:/usr/bin
+
+COMPONENT_PREP_ACTION=	cd $(@D)  && NOCONFIGURE=1 ./autogen.sh
+
+CONFIGURE_OPTIONS+= --sysconfdir=/etc
+CONFIGURE_OPTIONS+= --libexecdir=/usr/lib/mate
+CONFIGURE_OPTIONS+= --disable-static
+CONFIGURE_OPTIONS+= --disable-scrollkeeper
+CONFIGURE_OPTIONS+= --localstatedir=/var/lib
+CONFIGURE_OPTIONS+= --with-gtk=2.0
+CONFIGURE_OPTIONS+= --with-default-wm
+
+PYTHON_VERSION = 2.7
```

Comment:
mate-session-manager doesn't check for python, so this line and the next one are not needed here